### ISSUE TITLE
[0.9.0 Staging-1542] - Cannot make effective laws to subsidise dirt roads

### ISSUE THREAD

**Issue by trickenso** [StrangeLoopGames/EcoIssues#16249]
There is a law action referring to tamping roads with a road tool, and you might want to award someone money for doing this.
The issue comes from not being able to prevent the removal of a dirt road. It is impractical to prevent digging dirt blocks, as you might need dirt to be cleared to create a flat surface (or remove the surface for stone or asphalt roads), but you then open an exploit loop where a player could tamp a road for the subsidy, dig the dirt block, replace it, and repeat for infinite treasury cash.

Ideally, there would be a specific way of legislating dig actions on dirt road blocks - to prevent or tax these actions, perhaps.

TL;DR - Without the ability to legislate on removal of dirt road blocks, it is impossible to safely subside the addition of dirt road blocks.
